js/scripts.js

```diff
@@ -533,3 +579,12 @@ var mapStyles = [
         }]
     }
 ];
+
+// Only slightly scuffed solution to find a school from a list of institutions based on lat and lang
+function findSchoolFromLatLng(institutions, position) {
+    for (school in institutions) {
+        if(institutions[school].position.lat.toFixed(2) == position.lat.toFixed(2) && institutions[school].position.lng.toFixed(2) == position.lng.toFixed(2)) {
```

Comment:
Hmm. This probably works fine for now, but I think it would be a _tad_ cleaner to find the euclidean distance and check that it's below a certain threshold rather than check that these somewhat arbitrary string conversions result in the same string.